Libraries/LibCompress/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(SOURCES
     PackBitsDecoder.cpp
     Zlib.cpp
     Gzip.cpp
+    GenericZlib.cpp
```

Comment:
Could you reorder these to be in alphabetical order? (Yeah, `Gzip.cpp` was already in the wrong position)